src/app.module.ts

```diff
@@ -15,7 +18,7 @@ import * as path from 'path';
     ),
     FirebaseModule,
     AuthModule,
-    MongooseModule.forRoot('mongodb://localhost/nest'),
+    MongooseModule.forRoot(process.env.MONGOURL),
```

Comment:
create `.env.example` and put the environment variables with their example values.
like done here: https://github.com/codeforcauseorg/edu-client/blob/main/.env.example

Reply:
ok..
sure

Reply:
@Abhishek-kumar09 Done.. thanks for the review